database/sqlite/sqlite_aclk_alert.c

```diff
@@ -685,8 +687,8 @@ void aclk_start_alert_streaming(char *node_id, bool resets)
 }
 
 #define SQL_QUEUE_REMOVE_ALERTS "INSERT INTO aclk_alert_%s (alert_unique_id, date_created, filtered_alert_unique_id) " \
-                                "SELECT unique_id alert_unique_id, UNIXEPOCH(), unique_id alert_unique_id FROM health_log_%s " \
-                                "WHERE new_status = -2 AND updated_by_id = 0 AND unique_id NOT IN " \
+                                "SELECT unique_id alert_unique_id, UNIXEPOCH(), unique_id alert_unique_id FROM health_log " \
+                                "WHERE host_id = @host_id AND new_status = -2 AND updated_by_id = 0 AND unique_id NOT IN " \
                                 "(SELECT alert_unique_id FROM aclk_alert_%s) " \
                                 "AND config_hash_id NOT IN (select hash_id from alert_hash where warn is null and crit is null) " \
                                 "ORDER BY unique_id ASC " \
```

Comment:
You organize `unique_id` as ASC in the external query, but in the inner you did not organize them, probably, ordering both will simplify database's life.